setup

```diff
@@ -0,0 +1,84 @@
+whoisthis="Eric Harrison"  #your name, used for Git config
+youremail="ericmharrison@gmail.com"  #your email, used for Git config
+
+#update package list and upgrade all software
+sudo apt-get update 2>/dev/null && sudo apt-get -fy full-upgrade
+sudo apt -fy install software-properties-common apt-transport-https wget
+
+#check for git
+hasgit=$(apt list git --installed 2>/dev/null)
```

Comment:
this is a great way to know apt is sure that it has installed git, but as you just saw with Slack & snap... these package managers cant always be trusted. Consider running a git command, like version, to additionally verify your install

Reply:
i'm planning to eventually make it do as much through git as possible.  main goal right now is just getting things working